### PR TITLE
Update ember, ember-{cli,data} to 2.8

### PR DESCRIPTION
**Pull request by piotrpalek**

### CODE REVIEW

**Comment by stefanpenner** [comment]
nice!

**Comment by piotrpalek** [comment]
@stefanpenner I assume you saw my (mistakely, branched from the wrong place) pushed service worker commits and don't just get excited by this version bump :D I've since corrected this and force pushed to remove them.

File: package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "ember-api-actions": "~0.1.4",
     "ember-autoresize": "0.5.16",
     "ember-browserify": "1.1.11",
-    "ember-cli": "2.7.0",
+    "ember-cli": "^2.8.0",
```
Comment (Gaurav0) [inline package.json]: Please keep version of ember-cli locked. (No carat)

Comment (piotrpalek) [inline package.json]: @Gaurav0 fixed, I blame `ember install`

**Pull request merged**